test/coverlet.core.tests/Samples/Samples.cs

```diff
@@ -4,6 +4,8 @@
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
 using System.Linq;
+using System.Reflection;
```

Comment:
These using seem unuseful...can we remove?

Reply:
Done